clerk/templates_test.go

```diff
@@ -189,6 +189,7 @@ func TestTemplatesService_Preview_happyPath(t *testing.T) {
 const dummyTemplateJSON = `{
     "object": "template",
     "slug": "derp",
+	"resource_type": "user",
```

Comment:
:upside_down_face:

Reply:
Don't know why this is happening.

Looks fine locally in VS Code & Goland. 🤷